charts/prometheus-node-exporter/values.yaml

```diff
@@ -468,3 +468,246 @@ extraManifests: []
   #     name: prometheus-extra
   #   data:
   #     extra-data: "value"
+
+# prometheus process exporter configuration
+ProcessExporter:
+  enabled: false
+  nameOverride: "prometheus-process-exporter"
```

Comment:
The name of the container and daemonset should be `process-exporter`, `node-exporter` instead of adding `prometheus-` prefix to these names

Reply:
> The name of the container and daemonset should be `process-exporter`, `node-exporter` instead of adding `prometheus-` prefix to these names just like the yaml one https://github.com/kubesphere/kse-installer/blob/kse-3.x/roles/ks-monitor/templates/node-exporter-daemonset.yaml.j2#L9

@frezes what do you think?

Reply:
nameOverride does not affect the container name in the daemonset. As for the name of daemoset, according to the definition of chart it should be: {{ include "prometheus-node-exporter.fullname" . }}, like: release-name-prometheus-node-exporter
```
- name: process-exporter
  image: "ncabatoff/process-exporter:0.5.0"
  imagePullPolicy: IfNotPresent
  args:
    - --procfs=/host/proc
    - --config.path=/var/process-exporter/config.yml
    - --web.listen-address=0.0.0.0:9202
  resources:
            {}
  volumeMounts:
    - name: proc
      mountPath: /host/proc
      readOnly:  true
    - name: config
      mountPath: /var/process-exporter
```
```
- name: calico-exporter
  env:
    - name: NODENAME
      valueFrom:
        fieldRef:
          fieldPath: spec.nodeName
  image: "kubesphere/calico-exporter:v0.2.0"
  imagePullPolicy: IfNotPresent
  args:
    - --web.listen-address=127.0.0.1:9093
    - --collector.enable-collectors=bgp
  resources:
      limits:
        cpu: 1
        memory: 500Mi
      requests:
        cpu: 102m
        memory: 180Mi
  volumeMounts:
    - name: var-run-calico
      mountPath: /var/run/calico
```